snap/local/bin/disable-updater

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+if [ ! -d "$SNAP_USER_DATA"/.config/discord ]; then
+    mkdir -p "$SNAP_USER_DATA"/.config/discord
+fi
+
+CONFIG_FILE="$SNAP_USER_DATA"/.config/discord/settings.json
```

Comment:
Ulikely to happen, but if `$XDG_CONFIG_HOME` is set, then it would be used, so I'd instead use:

```suggestion
CONFIG_DIR=${XDG_CONFIG_HOME:-$SNAP_USER_DATA/.config}/discord
mkdir -p "$CONFIG_DIR"

CONFIG_FILE="$CONFIG_DIR"/settings.json
```

Reply:
Nice, thanks!